vue/src/vueTextMask.js

```diff
@@ -1,24 +1,19 @@
 import createTextMaskInputElement from '../../core/src/createTextMaskInputElement.js'
 
-export default {
-  textMaskInputElement: null,
-
-  inputHandler({target: {value}}) {
-    return this.textMaskInputElement.update(value)
-  },
+let textMaskInputElement = null
 
-  bind() {
-    let options = this.vm[this.expression] || {}
-    options.inputElement = this.el
-    this.textMaskInputElement = createTextMaskInputElement(options)
+const inputHandler = ({target: {value}}) => {
+  return textMaskInputElement.update(value)
+}
 
-    this.inputHandler = this.inputHandler.bind(this)
-    this.el.addEventListener('input', this.inputHandler)
+export default {
+  bind(el, binding, vnode) {
+    let options = binding.value || {}
+    options.inputElement = el
+    textMaskInputElement = createTextMaskInputElement(options)
+    el.addEventListener('input', inputHandler)
   },
-
   unbind() {
-    this.el.removeEventListener('input', this.inputHandler)
+    el.removeEventListener('input', inputHandler)
   }
 }
-
-export {default as conformToMask} from '../../core/src/conformToMask.js'
```

Comment:
We need to export `conformToMask` as documented [here](https://github.com/text-mask/text-mask/blob/master/componentDocumentation.md#included-conformtomask).